src/org/omegat/filters2/text/bundles/ResourceBundleFilter.java

```diff
@@ -146,50 +142,6 @@ public Instance[] getDefaultInstances() {
                 TFP_NAMEONLY + "_" + TFP_TARGET_LOCALE + "." + TFP_EXTENSION) };
     }
 
-    /**
-     * Creates a reader of an input file.
-     * <p>
-     * Override because of keep buggy behavior in OmegaT 5.7.1 or before. It set
-     * default encoding US-ASCII but Java standard InputStreamReader class
-     * wrongly accept non-ASCII characters as-is.
-     * </p>
-     *
-     * @param inFile
-     *            The source file.
-     * @param inEncoding
-     *            Encoding of the input file, if the filter supports it.
-     *            Otherwise null.
-     * @return The reader for the source file
-     * @throws IOException
-     *             If any I/O Error occurs upon reader creation
-     */
-    @Override
-    public BufferedReader createReader(File inFile, String inEncoding) throws IOException {
-        Charset charset;
-        if (inEncoding != null) {
-            charset = Charset.forName(inEncoding);
-        } else {
-            charset = Charset.defaultCharset();
-        }
```

Comment:
Here we finally change a buggy behavior. In previous versions, the `ResourceBundleFilter ` can read platform default charset,  such as Windows-1251, that is not compliant with Java standard.